content/news/2023-02-20-global-team-recruiting/index.en.md

```diff
@@ -0,0 +1,104 @@
+We think of ourselves as one larger Global Team comprised of many smaller teams that address specific organizational needs.
+We want to build a diverse Global Team with good representation of all community members.
+
+# Timeline (approximate)
+
+-   date1-date2: Google form is open to submissions (2 weeks)
+
+-   date3-date4: Global team reviews submissions (2 weeks)
+
+-   date5-date6: Target date to notify individuals of selection.
+    (1 week)
+
+-   date7-date8: Invitation to meet with 1-3 global team members to clarify any questions or expectations to ensure a good match for all parties.
+    (2 weeks)
+
+# Selection considerations
+
+We will consider various aspects with the aim of increasing the diversity of the team.
+The following are not an exhaustive list of requirements but rather an illustration of the various paths possible.
+A new team member is not expected to satisfy all criteria.
+
+-   Geographical location
+
+-   Demographic characteristics like ethnicity, age, and gender identity
+
+-   Background and experience such as education, skills, and knowledge \<--Note: if we keep this here then should google form should be ammended to capture this??--\>
+
+-   R-Ladies/R community involvement
+
+# Benefits
+
+By participating in R-Ladies you are helping other women or gender minorities by creating a safe space to learn, teach, and develop, as well as build out a support network and contacts.
+
+Some of the personal benefits that we think come with joining the global team and volunteering time are gaining experience in leadership, asynchronous remote work, international team work, not-for-profit open-source work, and work in a women(+) only team and organization.
+
+Depending on the role, there also may be opportunities for technical upskilling.
+For example, some members of the Global team use R, reproducible reports, git/GitHub, GitHub Actions, Hugo website development, and database integrations.
+(Knowledge of these tools is not a requirement for selection; the Global team will provide training on processes specific to your role.)
+
+# Commitment
+
+We ask those seeking consideration to commit to one year of 1 to 4 hours a week for service to the Global Team.
+Of course, vacations and exceptions may occur.
+
+# Positions
+
+Below is a list of all current global team operations. All teams are listed, however
+not all teams are currently recruiting. Please refer to this table when submitting
+your interest. 
+
+In addition, applying to a specific team does not mean your contributions are limited to that
+team. Individual teams often solicit and receive feedback or assistance from other
+members of the Global Team.
+
+
+|Team|Brief responsibilities|Seeking # members|
+|:----|:----|:----|
+|Social Media|Post regularly on social media platforms (e.g., Twitter, Mastodon, LinkedIn, etc.) and create a regular newsletter|3|
```

Comment:
It is best if we split social media from newsletters. Are different activities and have their own process and audience.

Reply:
i believe @maelle liked the idea of having that team together, what do you think?

Reply:
What if it were called "external communication"? :thinking:

Reply:
I like having these together in a group, as they'd likely have close connection to get their tasks done.

Reply:
Communications sounds good to me!